src/Config/CS_SDK.props

```diff
@@ -1,14 +1,15 @@
 <Project>
   <PropertyGroup>
+    <Platforms>AnyCPU;NET60_Windows;NET60_Linux;</Platforms>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <PlatformTarget >AnyCPU</PlatformTarget>
```

Comment:
`Platform` can have any value. It is user-defined.
`PlatformTarget` must have a valid value of AnyCPU, x86, x64, ARM,

Reply:
maybe not the time for it, but we don't really support x86 anymore.